src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -452,6 +457,10 @@ class DockerCreateContainer extends DockerExistingImage {
         if(macAddress.getOrNull()) {
             containerCommand.withMacAddress(macAddress.get())
         }
+
+        if(tmpfs.getOrNull()) {
+            containerCommand.hostConfig.withTmpFs(tmpfs.get().collectEntries { [it, ""] })
```

Comment:
Wouldn't be better to allow users to provide also mount options?

Reply:
you mean in addition to tmpfs or in place of that?

Reply:
In addition. Then the new property should be of type `Porperty<Map<String, String>>` and have a JavaDoc explaining what are the keys and values.

Reply:
multiple mounts can be specified, though, so it also needs to be a list. i guess the map you had in mind was of the kv pairs that get passed as options to mount?

Reply:
I suggest you implement a more complex data structure to cover all possible cases similar to what Docker Java does. Please introduce your own classes if you can so that the Docker Java classes don't bleed into the public API of the plugin if possible.

Reply:
@thyming Are you still planning to finish this PR?

Reply:
I started down the path of implementing functionality equivalent to docker cli `--mount` but found that there was quite a lot of parsing logic of the options not implemented in docker-java (as it is for ports, volumes, etc) that I didn't really have time to investigate/implement. A quick hack at it led to nasty mapping code between strings and docker-api's enums without bringing in jackson as a direct dependency.

Reply:
Is it correct to assume that you won't have the time to follow up? I am going to close the PR soon if that's the case as the solution is not sufficient. Thanks again for your contribution.

Reply:
No, I don't have time to work through a full support of the mount flag. However, I'm not sure why it's not worth merging the tmpfs option, which, while less fully featured than the mount option, is still an element of the CLI and Java API and so seems reasonable to expose here.

Reply:
The data structure should reflect something we can enhance in the future. If we want to add something new to what you propose then this would likely change the public API of the plugin and therefore break anyone using the new property.

It's not a matter of introducing this configuring option in general. It's a matter of maintainability and future enhancement without breaking existing consumers.

Reply:
In that case, I will close it.